### DISCUSSION TITLE
PDF embeds keep automatically zooming out

### DISCUSSION THREAD

**Discussion by JennaSmu** [RyotaUshio/obsidian-pdf-plus#280] [Q&A]
Hi, I love PDF++ but every time I put in a PDF embed, it automatically starts zooming out whenever I make a change to a note which is super annoying. I found the same issue I am experiencing described here in this past issue: https://github.com/RyotaUshio/obsidian-pdf-plus/issues/137 

While this issue was marked as resolved, I am still having this issue. Is there a certain setting I need to change? 

I have updated my PDF++ to the most recent version today and I am still having this issue and it is super annoying. Any advice or tips would be greatly appreciated.

**Comment by RyotaUshio** [comment]
First time I heard that this re-appeared. Can you file a bug report so that I can reproduce it in a clean vault and understand what's going on? Please include all necessary information such as the exact PDF file and markdown content.

**Reply by JennaSmu** [reply]
Hey, so I uninstalled it and reinstalled it on my computer as I couldn't replicate the problem in the sandbox vault. It's fixed now!
